build.gradle

```diff
@@ -378,9 +377,6 @@ project ('spring-kafka-test') {
 		api 'org.junit.platform:junit-platform-launcher'
 		optionalApi "org.hamcrest:hamcrest-core:$hamcrestVersion"
 		optionalApi "org.mockito:mockito-core:$mockitoVersion"
-		optionalApi ("junit:junit:$junit4Version") {
```

Comment:
This is not correct.
We are talking about deprecation of the utility, but not total removal.
Therefore dependency and deprecated classes must still be here.
Again: we develop library, so whatever we do here might break target projects using our library.

Removing it temporary locally is a good way to check if we still have any JUnit 4 tests left. 😄